src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -3587,6 +3597,12 @@ type Stoptime {
     """The stop where this arrival/departure happens"""
     stop: Stop
 
+    """
+    The sequence of the stop in the trip. This is not required to start from 0 or be consecutive - any
+    increasing integer sequence along the stop sequence is valid.
+    """
+    sequence: Int
```

Comment:
Should it be 'sequenceNr' ? 

I like that you do not say that this is the GTFS stop sequence number - this means we can cange this to e.g stopPosInPattern, if we use GTFS and Netex data. 

Should we mention that this is not necessarily the GTFS seq.nr, and that it is provided to identify the stop within the pattern. It is "safe" to use for mapping between an itinerary leg and a pattern within a short time frame(seconds). It is not 100% safe since a RT update may alter the pattern.

Reply:
In the GTFS spec this is called `stopSequence` not sequence number: https://github.com/google/transit/blob/master/gtfs/spec/en/reference.md#stop_timestxt

That's why I went with the name.

I've updated the documentation in 4bd1342f8f3bac8ad8575e5e4365daeab22121ac

Reply:
Should we call it stopSequence here as well or sequence?

Reply:
The type that is on is called `StopTime` so should be called `StopTime#stopSequence`? I don't particularly like that.

Reply:
It would be nice if we do NOT link this to the GTFS `stopSequence`. Because that would make it possible to return a sequence/index and let the clients use this to match stops - but the underlaying datasource could be NeTEx or GTFS. So, using a different name than the name in the GTFS spec is desirable. We give the same guaranties as the GTFS stopSequence - but we should not guarantee it is the same as in the input. This make the system much more robust, and allow inserting stops in patterns RT. Then we can make this a none-nullable field as well.

Reply:
We ended up calling it `stopPattern`.